core/backend/src/ECSqlStatement.ts

```diff
@@ -320,36 +331,54 @@ export class ECSqlStatement implements IterableIterator<any>, IDisposable {
    * - [ECSQL row format]($docs/learning/ECSQLRowFormat) for details about the format of the returned row.
    * - [Code Samples]($docs/learning/backend/ECSQLCodeExamples#working-with-the-query-result)
    */
-  public getRow(): any {
-    const colCount: number = this.getColumnCount();
-    const row: object = {};
-    const duplicatePropNames = new Map<string, number>();
-    for (let i = 0; i < colCount; i++) {
-      const ecsqlValue = this.getValue(i);
-      if (!ecsqlValue.isNull) {
-        const propName: string = ECSqlStatement.determineResultRowPropertyName(duplicatePropNames, ecsqlValue);
-        const val: any = ecsqlValue.value;
-        Object.defineProperty(row, propName, { enumerable: true, configurable: true, writable: true, value: val });
-      }
+  public getRow(args?: ECSqlRowArg): any {
+    args = args ?? {};
+
+    // Set default rowFormat if not provided
+    if (args.rowFormat === undefined) {
+      args.rowFormat = QueryRowFormat.UseJsPropertyNames;
     }
-    return row;
+
+    const toRowOptions = {
+      classIdsToClassNames: true,
```

Comment:
why didnt we re-use [convertClassIdsToClassNames](https://www.itwinjs.org/reference/core-common/imodels/queryoptions/convertclassidstoclassnames/) instead of creating a new name?
this is breaking functionally and should've been opt in vs set as default

Reply:
ECSqlRowArg interface has only 1 optional property `rowFormat`. This is because ECSqlStatement always changed class IDs to class names in the past. To keep this working as before, I used the `classIdsToClassNames` set to true in the API.
CC: @khanaffan